navidrome/umbrel-app.yml

```
@@ -0,0 +1,59 @@
+manifestVersion: 1
+id: navidrome
+category: media
+name: Navidrome
+version: "0.52.5"
+tagline: 🎧 ☁️ Modern Music Server and Streamer compatible with Subsonic/Airsonic
+description: >-
+  Your Personal Streaming Service
+  Navidrome is an open source web-based music collection server and streamer.
+  It gives you freedom to listen to your music collection from any browser or mobile device. It's like your personal Spotify!
+
+  Features:
+
+  - Handles very large music collections
+
+  - Streams virtually any audio format available
+
+  - Reads and uses all your beautifully curated metadata
+
+  - Great support for compilations (Various Artists albums) and box sets (multi-disc albums)
+
+  - Multi-user, each user has their own play counts, playlists, favourites, etc...
+
+  - Very low resource usage
+
+  - Multi-platform, runs on macOS, Linux and Windows. Docker images are also provided
+
+  - Ready to use binaries for all major platforms, including Raspberry Pi
+
+  - Automatically monitors your library for changes, importing new files and reloading new metadata
+
+  - Themeable, modern and responsive Web interface based on Material UI
+
+  - Compatible with all Subsonic/Madsonic/Airsonic clients
+
+  - Transcoding on the fly. Can be set per user/player. Opus encoding is supported
+
+  - Translated to various languages
+
+developer: Navidrome
+website: https://www.navidrome.org/
+dependencies: []
+repo: https://github.com/navidrome/navidrome
+support: https://github.com/navidrome/navidrome/discussions
+port: 4533
+gallery:
+  - 1.jpg
+  - 2.jpg
+  - 3.jpg
+path: ""
+defaultUsername: ""
+defaultPassword: ""
+releaseNotes: >-
+
+  This is mostly a bug-fix version, with focus on some Jukebox issues, but also other general improvements and fixes.
+
+  Full release notes are available at https://github.com/navidrome/navidrome/releases/tag/v0.52.5
```

For initial app submissions release notes are not required. Those are being displayed to the user when there is a new update available.
```suggestion
releaseNotes: ""
```